cypress/integration/bookshelf_spec.js

```diff
@@ -0,0 +1,29 @@
+describe('BookshelfDisplay', () => {
+  beforeEach( () => {
+    cy.load()
+    cy.bookshelf()
+  })
+
+  it('Should display the navbar', () => {
+    cy.get('.nav').should('be.visible')
+      .get('nav > h1').should('contain', 'BookWorm')
+      .get('nav > ul > li').eq(0).should('contain', 'Home')
+      .get('nav > ul > li').should('contain', 'Favorites')
+  })
+
+  it('Should display bookcards based on a genre', () => {
+    cy.get('.bookshelf').should('be.visible')
+      .get('.bookCard').should('have.length', 2)
+      .get('.bookshelf > article').eq(0).should('contain', 'THE LAST THING HE TOLD ME')
+        .should('contain', 'Laura Dave')
+        .should('contain', 'Rank: 1')
+      .get('.bookshelf > article').eq(1).should('contain', 'SOOLEY')
+        .should('contain', 'John Grisham')
+        .should('contain', 'Rank: 2')
+  })
+
+  it('Should include the queryName in the URL', () => {
+     cy.url().should('eq', 'http://localhost:3000/bookshelf/combined-print-and-e-book-fiction')
+  })
+
+})
```

Comment:
Looks great, you are testing for everything we'd except. Not sure how we could enhance this,  I really like how you tested for the url, I didn't think of that! We can add a url test to the dashboard_spec.js as well.